docs/tools.rst

```diff
@@ -125,6 +125,9 @@ Forwarding
 `statsd`_
   A simple daemon for easy stats aggregation, developed by the folks at Etsy. A list of forks and alternative implementations can be found at <http://joemiller.me/2011/09/21/list-of-statsd-server-implementations/>
 
+`_Statusengine`_
```

Comment:
Build fails:

> /home/travis/build/graphite-project/graphite-web/docs/tools.rst:129: ERROR: Unknown target name: "_statusengine".

I think you need to get rid of the prefix underscore

Reply:
Thanks for the hint looks better now :)